### ISSUE TITLE
Permission issue during backup encryption on Openshift

### ISSUE THREAD

**Issue by jkaninda** [jkaninda/pg-bkup#54]
Permission denied during backup encryption on Openshift

```sh
2024/08/14 09:54:24 INFO: Backup database to Remote server
22024/08/14 09:54:24 INFO: Starting database backup...
32024/08/14 09:54:24 INFO: Connecting to database ...
42024/08/14 09:54:24 INFO: Successfully connected to database
52024/08/14 09:54:24 INFO: Backing up database...
62024/08/14 09:54:24 INFO: Database has been backed up
72024/08/14 09:54:24 INFO: Encrypting backup...
8gpg: Fatal: can't create directory '//.gnupg': Permission denied
92024/08/14 09:54:24 ERROR: Error during encrypting backup exit status 2
```

**Comment by jkaninda** [comment]
This issue will be fixed in the upcoming update.
Meanwhile, `GNUPGHOME` environment variable can be set to `/tmp` to fix the permission issue

```yaml
        env:
          - name: GNUPGHOME
            value: "/tmp"
```

**Comment by jkaninda** [comment]
Fixed in v1.2.2 #55

**Comment by jkaninda** [comment]
Fixed in v1.2.2 #55
